### PR TITLE
prettyclean 0.1.38 (new cask)

### PR DESCRIPTION
**Pull request by JaminYe**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/p/prettyclean.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+cask "prettyclean" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "0.1.38"
+  sha256 arm:   "1832d6ffdbe0fcbffe35c3b2f6ac4031e7e8ce93a7c564124914888ebfb43885",
+         intel: "bc7512ff7544d8522ebd1ac0c13e94da8661dd6a5772bda33d68a9145be73436"
+
+  url "https://downloads.jmotor.org/prettyclean/v#{version}/darwin-#{arch}/PrettyClean_#{version}_#{arch.gsub("86_", "")}.dmg",
+      verified: "downloads.jmotor.org/"
+  name "prettyclean"
```
Comment (bevanjkay) [inline Casks/p/prettyclean.rb:10]:
```suggestion
  name "PrettyClean"
```

File: Casks/p/prettyclean.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  desc "Easy to use Disk Cleanup Tools"
+  homepage "https://www.prettyclean.cc/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist do |items|
+      items["cc.prettyclean.PrettyClean"].short_version
+    end
+  end
```
Comment (bevanjkay) [inline Casks/p/prettyclean.rb:19]:
```suggestion
  livecheck do
    url "https://www.prettyclean.cc/en/download"
    regex(/href=.*?PrettyClean[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
  end
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Comment by reitermarkus** [comment]
@JaminYe, don't mark the requested changes as resolved when they are not.

File: Casks/p/prettyclean.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+cask "prettyclean" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "0.1.38"
+  sha256 arm:   "1832d6ffdbe0fcbffe35c3b2f6ac4031e7e8ce93a7c564124914888ebfb43885",
+         intel: "bc7512ff7544d8522ebd1ac0c13e94da8661dd6a5772bda33d68a9145be73436"
+
+  url "https://downloads.jmotor.org/prettyclean/v#{version}/darwin-#{arch}/PrettyClean_#{version}_#{arch.gsub("86_", "")}.dmg",
```
Comment (SMillerDev) [inline Casks/p/prettyclean.rb:8]: Why not name the arch differently then?

**Comment by bevanjkay** [comment]
@JaminYe Thank you. Just a minor change to flip which variable uses `arch` and which is "named" (simply to follow the more standard logic).

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**